internal/api/web/views/components/quiz_components/quiz_summary.templ

```diff
@@ -24,7 +24,6 @@ templ QuizSummary(summary users.UserQuizSummary) {
 				// for i := 0; i < 10; i++ {
 				for _, aq := range summary.AnsweredQuestions {
 					<li class="mb-2">
-						{ aq.QuestionText }
```

Comment:
what
why remove the question text? why is the summary struct affected in the first place, it is used strictly for quiz summary display. the question text is bnecesary there (according to figma)